packages/financial-templates-lib/test/price-feed/BalancerPriceFeed.js

```diff
@@ -1,24 +1,34 @@
+const { web3, getContract, network } = require("hardhat");
+const { assert } = require("chai");
 const { toWei, toBN } = web3.utils;
 const winston = require("winston");
 
-const { BalancerPriceFeed } = require("../dist/price-feed/BalancerPriceFeed");
-const { mineTransactionsAtTime, MAX_SAFE_JS_INT } = require("@uma/common");
-const { delay } = require("../dist/helpers/delay.js");
-const { getTruffleContract } = require("@uma/core");
+const { BalancerPriceFeed } = require("../../dist/price-feed/BalancerPriceFeed");
+const { mineTransactionsAtTimeHardhat, MAX_SAFE_JS_INT } = require("@uma/common");
+const { delay } = require("../../dist/helpers/delay.js");
 
-const BalancerMock = getTruffleContract("BalancerMock", web3);
-const Balancer = getTruffleContract("Balancer", web3);
+const BalancerMock = getContract("BalancerMock");
+const Balancer = getContract("Balancer");
 
-contract("BalancerPriceFeed.js", function (accounts) {
-  const owner = accounts[0];
+const getFutureTimestamp = async () => {
```

Comment:
might want to comment on this function and why its used

Reply:
Great point. Will do.

Reply:
Done!